rules/feature/targettype.yaml

```diff
@@ -96,3 +96,11 @@
 		RequiresForceFire: true
 	Targetable@TargetTypeNoAutoTarget:
 		TargetTypes: NoAutoTarget
+
+^FeatureTargettypeFloraAir:
```

Comment:
I used it to stop turrets and units from targeting the Units. Feel free to rename it if you dislike it.